app/models/supervisor_script_executors/supervisor_script_executors.rb

```diff
@@ -0,0 +1,33 @@
+Dir[Rails.root.join('supervisor_scripts', 'executors', '*_executor.rb').to_s].each {|file| require file}
+require_relative 'abstract_supervisor_script_executor'
+##
+# This class translates simulation scripts id to their executor
+class SupervisorScriptExecutors
+  private
```

Comment:
czy to private Ci działa?

zobacz: http://stackoverflow.com/questions/4952980/creating-private-class-method

druga sprawa: przynajmniej ja preferuję umieszczanie metod prywatnych poniżej publicznych i używanie "private" z wypisywaniem sygnatur metod (tak jak w pierwszej odpowiedzi na stacku); niestety w starym kodzie używane są private bez wypisywania sygnatur co mogę objaśnić kiedyś na spotkaniu